dashboard/test/models/activity_test.rb

```diff
@@ -1,5 +1,6 @@
 require 'test_helper'
 
+require 'async_progress_handler'
```

Comment:
There's a discipline called IWYU (include what you use) that I try to follow.  (Though I'm sure I don't do it absolutely consistently!)  The advantages are that it makes dependencies more obvious and also that it this more likely to work even if this code happens to be used in a place that doesn't include async_progress_handler.